src/utils/Parser.H

```diff
@@ -29,15 +29,13 @@ queryWithParser (const amrex::ParmParse& pp, char const * const str, T& val);
 namespace Parser
 {
     // Cache for evaluated constants
-    static std::map<std::string, double> my_constants_cache{};
+    extern std::map<std::string, double> my_constants_cache;
```

Comment:
I think these global variables should be neither `static` nor `extern` but simple (global) variables instead...?

I am no expert in globals tbh, because reasons :D
I would probably wrap them in a `struct` instead of a namespace and then declare them `static` (which is essentially the same).

cc @WeiqunZhang

Reply:
I agree with Weiqun, possibly `inline` would have worked but I didn’t try this.

My Question is why it worked before this PR in release mode, but not in debug mode (where there where multiple instances of hipace_constants)